keycloak/keycloak-admin-ui
# realm roles: associated roles modal (roles list)

## Motivation


#284 , https://issues.redhat.com/browse/APPDUX-746

## Brief Description


Builds out modal for adding associated roles.

## Verification Steps


1. Go to Realm Roles section.
2. Click on one of the roles in the list.
3. Go to the Actions dropdown in the upper right.
4. Click "Add associated roles"
5. Verify that the modal renders and displays the list of existing realm roles. 

## Checklist:


## Additional Notes
